tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -191,6 +210,12 @@ rootfs-nvidia-gpu-confidential-initrd-tarball: agent-tarball busybox-tarball pau
 shim-v2-tarball:
 	${MAKE} $@-build
 
+# The shim-v2 build for aarch64 needs the kernel-headers tar file from kernel-cca-confidential kernel.
```

Comment:
Description about why add this:
The kernel-headers are needed mandatorily for building shim-v2. In Kata runtime we need to get the const values from new version kvm.h(https://github.com/kata-containers/kata-containers/pull/10664/files#diff-c01cf30f6f3d0de0ea77d01a7acdc8c6dcc514f8ef1e19ccbe5684eabebc2b6cR10) which is currently not included into the kernel build image.

So, I set a requisition here to force building kernel-cca-confidential, and then we assume that the kernel headers will be packaged after the building process. This is only needed for Arm64.
Then, we can comes to build shim-v2 without any errors.